Makefile

```
@@ -391,68 +394,63 @@ run-tests-cluster-vst-1.1-ssl:
 
+
+COMMON_DOCKER_CMD_PARAMS = \
+	--name=$(TESTCONTAINER) \
+	$(TEST_NET) \
+	-e TEST_ENDPOINTS=$(TEST_ENDPOINTS) \
+	-e TEST_NOT_WAIT_UNTIL_READY=$(TEST_NOT_WAIT_UNTIL_READY) \
+	-e TEST_AUTHENTICATION=$(TEST_AUTHENTICATION) \
+	-e TEST_JWTSECRET=$(TEST_JWTSECRET) \
+	-e TEST_MODE=$(TEST_MODE) \
+	-e TEST_BACKUP_REMOTE_REPO=$(TEST_BACKUP_REMOTE_REPO) \
+	-e TEST_BACKUP_REMOTE_CONFIG='$(TEST_BACKUP_REMOTE_CONFIG)' \
+	-e TEST_DEBUG='$(TEST_DEBUG)' \
+	-e TEST_ENABLE_SHUTDOWN=$(TEST_ENABLE_SHUTDOWN) \
+	-e ENABLE_DATABASE_EXTRA_FEATURES=$(ENABLE_DATABASE_EXTRA_FEATURES) \
+	-e GODEBUG=tls13=1 \
+	-e CGO_ENABLED=$(CGO_ENABLED)
+
+
 # Internal test tasks
 __run_tests: __test_debug__ __test_prepare __test_go_test __test_cleanup
 
+
+ALL_DOCKER_CMD_PARAMS=\
+	$(COMMON_DOCKER_CMD_PARAMS) \
+	-e TEST_CONNECTION=$(TEST_CONNECTION) \
+	-e TEST_CVERSION=$(TEST_CVERSION) \
+	-e TEST_CONTENT_TYPE=$(TEST_CONTENT_TYPE) \
+	-e TEST_PPROF=$(TEST_PPROF) \
+	-e TEST_REQUEST_LOG=$(TEST_REQUEST_LOG) \
+	-e TEST_DISALLOW_UNKNOWN_FIELDS=$(TEST_DISALLOW_UNKNOWN_FIELDS) \
+	-v "${ROOTDIR}":/usr/code ${TEST_RESOURCES_VOLUME} \
+	-w /usr/code/
+
 __test_go_test:
-	$(DOCKER_CMD) \
-		--name=$(TESTCONTAINER) \
-		$(TEST_NET) \
-		-v "${ROOTDIR}":/usr/code ${TEST_RESOURCES_VOLUME} \
-		-e TEST_ENDPOINTS=$(TEST_ENDPOINTS) \
-		-e TEST_NOT_WAIT_UNTIL_READY=$(TEST_NOT_WAIT_UNTIL_READY) \
-		-e TEST_AUTHENTICATION=$(TEST_AUTHENTICATION) \
-		-e TEST_JWTSECRET=$(TEST_JWTSECRET) \
-		-e TEST_CONNECTION=$(TEST_CONNECTION) \
-		-e TEST_CVERSION=$(TEST_CVERSION) \
-		-e TEST_CONTENT_TYPE=$(TEST_CONTENT_TYPE) \
-		-e TEST_PPROF=$(TEST_PPROF) \
-		-e TEST_MODE=$(TEST_MODE) \
-		-e TEST_BACKUP_REMOTE_REPO=$(TEST_BACKUP_REMOTE_REPO) \
-		-e TEST_BACKUP_REMOTE_CONFIG='$(TEST_BACKUP_REMOTE_CONFIG)' \
-		-e TEST_DEBUG='$(TEST_DEBUG)' \
-		-e TEST_ENABLE_SHUTDOWN=$(TEST_ENABLE_SHUTDOWN) \
-		-e TEST_REQUEST_LOG=$(TEST_REQUEST_LOG) \
-		-e TEST_DISALLOW_UNKNOWN_FIELDS=$(TEST_DISALLOW_UNKNOWN_FIELDS) \
-		-e ENABLE_DATABASE_EXTRA_FEATURES=$(ENABLE_DATABASE_EXTRA_FEATURES) \
-		-e GODEBUG=tls13=1 \
-		-e CGO_ENABLED=$(CGO_ENABLED) \
-		-w /usr/code/ \
-		$(DOCKER_RUN_CMD) && echo "success!" || ( \
-			echo "failure! \n\nARANGODB-STARTER logs:"; \
-			docker logs ${TESTCONTAINER}-s; \
-			echo "\nARANGODB logs:"; \
-			docker ps -f name=${TESTCONTAINER}-s- -q | xargs -L 1 docker logs; \
-			echo "\nV1 Tests with ARGS: TEST_MODE=${TEST_MODE} TEST_AUTH=${TEST_AUTH} TEST_CONTENT_TYPE=${TEST_CONTENT_TYPE} TEST_SSL=${TEST_SSL} TEST_CONNECTION=${TEST_CONNECTION} TEST_CVERSION=${TEST_CVERSION}\n\n"; \
-			exit 1)
+	$(DOCKER_CMD) $(ALL_DOCKER_CMD_PARAMS) $(DOCKER_RUN_CMD) \
+	&& echo "success!" \
+	|| ( \
+		$(foreach var,$(sort $(.VARIABLES)),$(eval export $(var)=$($(var)))) \
+		MAJOR_VERSION=1 . ./test/on_failure.sh \
+	)
+
+			
 # Internal test tasks
 __run_v2_tests: __test_v2_debug__ __test_prepare __test_v2_go_test __test_cleanup
 
+ALL_DOCKER_CMD_V2_PARAMS=\
```

```suggestion
ALL_DOCKER_CMD_V2_PARAMS=\
```
DOCKER_CMD_V2_PARAMS=\